pnpm-lock.yaml

```diff
@@ -8957,6 +8967,41 @@ snapshots:
       - supports-color
       - ts-node
 
+  '@jest/core@29.7.0(ts-node@10.9.2(@swc/core@1.12.0(@swc/helpers@0.5.17))(@types/node@22.16.0)(typescript@5.8.3))':
```

Comment:
maybe we can run `pnpm dedupe` to reduce duplicated dependencies

Reply:
Done